amundsen_application/static/js/index.tsx

```diff
@@ -40,7 +40,7 @@ ReactDOM.render(
   <DocumentTitle title="Amundsen - Data Discovery Portal">
     <Provider store={store}>
       <Router history={BrowserHistory}>
-        <main id="main">
+        <div id="main">
```

Comment:
Why the change given that this was recently intentionally changed from `div` to `main`

Reply:
Yep, that first change was a shortcut to have always a main. The problem is that aside, footer and header can't go inside the main element.
With this PR, the markup is in a much better place, although it took some extra work.

Reply:
Got it. Noted that the `main` is now inside the pages themselves.